Command/CleanCommand.php

```diff
@@ -45,7 +47,7 @@ protected function configure()
         parent::configure();
 
         $this
-            ->setName('fos:oauth-server:clean')
+            ->setName(self::$defaultName)
```

Comment:
This can be removed as it's automatically done by the base command constructor starting from Symfony 3.4 and we now support 4.4 and upper in master.